server/proxy-server.js

```diff
@@ -1,14 +1,22 @@
 const express = require('express');
 const parser = require('body-parser');
+const _ =require('underscore');
+const fs = require('file-system');
```

Comment:
nice this works, node also has their own fs and can just require ('fs'). maybe there is something better on this one file-system package?